distributed/deploy/local.py

```diff
@@ -30,6 +30,8 @@ class LocalCluster(SpecCluster):
     ----------
     n_workers: int
         Number of workers to start
+    memory_limit: str
```

Comment:
```suggestion
    memory_limit: str, float, int, or None, default "auto"
```
Here you can see all the many confusing possibilities for the `memory_limit` parameter: https://github.com/dask/distributed/blob/caf51895f63de4338a5a0c7862ecba0b8f7eb002/distributed/worker_memory.py#L383-L405

We should mention these options in the docstring, maybe like
```
* If None or 0, no limit is applied.
* If "auto", the total system memory is split evenly between the workers.
* If a float, that fraction of the system memory is used *per worker*.
* If a string giving a number of bytes (like ``"1GiB"``), that amount is used *per worker*.
* If an int, that number of bytes is used *per worker*.
```

Reply:
Oh that's good to know!